docs/recipes/php-cs-fixer/PrettierPHPFixer.php

```diff
@@ -58,37 +58,13 @@ public function supports(SplFileInfo $file) {
     }
 
     /**
-     * {@inheritdoc}
-     */
-    private function applyFix(SplFileInfo $file, Tokens $tokens) {
-        $tmpFile = $this->getTmpFile($file);
-        exec("yarn exec -- prettier --write $tmpFile");
-
-        $content = file_get_contents($tmpFile);
-        $tokens->setCode($content);
-
-        (new Filesystem())->remove($tmpFile);
-    }
-
-    /**
-     * Create a Temp file with the same content as given file.
-     *
-     * @param SplFileInfo $file file to be copied
-     *
-     * @return string tmp file name
+     * @param SplFileInfo $file
+     * @param Tokens      $tokens
      */
-    private function getTmpFile(SplFileInfo $file): string {
-        $fileSys = new Filesystem();
-        $tmpFolderPath = __DIR__.DIRECTORY_SEPARATOR.'tmp';
-        $fileSys->mkdir($tmpFolderPath);
-
-        $tmpFileName = str_replace(
-            array(DIRECTORY_SEPARATOR, ':'),
-            '_',
-            $file->getRealPath()
-        );
-        $tmpFilePath = $tmpFolderPath.DIRECTORY_SEPARATOR.'__'.$tmpFileName;
-        $fileSys->copy($file->getRealPath(), $tmpFilePath, true);
-        return $tmpFilePath;
+    private function applyFix(SplFileInfo $file, Tokens $tokens): void
+    {
+        exec("yarn exec -- prettier $file", $prettierOutput);
+        $code = implode("\r\n", $prettierOutput);
```

Comment:
I'd suggest use `PHP_EOL` over `\r\n` for this case.

Reply:
First I wanted to use `PHP_EOL` too, but this does not work because it is defined as `"\n"`. However, it seems like PHP CS Fixer requires the carriage return to be present. Else the output will be empty.

Reply:
That's interesting, php-cs-fixer worked for me somehow. 
May I know if you're using macos or linux or windows? 
Setting CRLF as EOL for everyone can cause issue for some if they don't have git auto CRLF to LF conversion set correctly

Reply:
Ok, you're right, `PHP_EOL` does work as expected.  For some reason the output was empty when I tested first but it worked now :+1: .

Reply:
@Billz95 Can I go ahead and merge? :upside_down_face:

Reply:
lgtm, go ahead, thanks :D